apps/studio/data/database/foreign-key-constraints-query.ts

```diff
@@ -107,19 +129,32 @@ export const useForeignKeyConstraintsQuery = <
       projectRef,
       connectionString,
       sql: getForeignKeyConstraintsQuery({ schema }),
-      queryKey: ['foreign-key-constraints'],
+      queryKey: ['foreign-key-constraints', schema],
```

Comment:
nit: `[schema, 'foreign-key-constraints']`? if we wanna invalidate foreign key constraints (assumingly at a schema level) it'll be easier